### PR TITLE
Update terraform docs for drift and formatting

### PR DESCRIPTION
**Pull request by h-romeus**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/d/project_config.html.markdown [hunk 1]
```diff
@@ -56,9 +56,9 @@ Nested schema for **definition**:
 		* `instance_location` - (String) The location of the compliance instance.
 		  * Constraints: The maximum length is `12` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(us-south|us-east|eu-gb|eu-de)$/`.
 		* `profile_name` - (String) The name of the compliance profile.
-		  * Constraints: The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^(?!\\s)(?!.*\\s$)[^`<>\\x00-\\x1F]*$/`.
+		  * Constraints: The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^(?!\\s)(?!.*\\s$)[^<>\\x00-\\x1F]*$/`.
 	* `description` - (String) A project configuration description.
-	  * Constraints: The default value is ``. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
+	  * Constraints: The default value is `[]`. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
```
Comment (riccardo-angelilli) [inline website/docs/d/project_config.html.markdown:61]: the default value is not empty array `[]`, but empty string `''`

File: website/docs/d/project_environment.html.markdown [hunk 2]
```diff
@@ -56,12 +56,12 @@ Nested schema for **definition**:
 		* `instance_location` - (String) The location of the compliance instance.
 		  * Constraints: The maximum length is `12` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(us-south|us-east|eu-gb|eu-de)$/`.
 		* `profile_name` - (String) The name of the compliance profile.
-		  * Constraints: The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^(?!\\s)(?!.*\\s$)[^`<>\\x00-\\x1F]*$/`.
+		  * Constraints: The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^(?!\\s)(?!.*\\s$)[^<>\\x00-\\x1F]*$/`.
 	* `description` - (String) The description of the environment.
-	  * Constraints: The default value is ``. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
+	  * Constraints: The default value is `[]`. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
```
Comment (riccardo-angelilli) [inline website/docs/d/project_environment.html.markdown:61]: the default value is not empty array `[]`, but empty string `''`

File: website/docs/r/project_config.html.markdown [hunk 3]
```diff
@@ -66,7 +66,7 @@ Nested schema for **definition**:
 	* `name` - (Required, String) The configuration name. It is unique within the account across projects and regions.
 	  * Constraints: The maximum length is `128` characters. The minimum length is `1` character. The value must match regular expression `/^[a-zA-Z0-9][a-zA-Z0-9-_ ]*$/`.
 	* `resource_crns` - (Optional, List) The CRNs of resources associated with this configuration.
-	  * Constraints: The list items must match regular expression `/(?!\\s)(?!.*\\s$)^(crn)[^'"`<>{}\\s\\x00-\\x1F]*/`. The maximum length is `110` items. The minimum length is `0` items.
+	  * Constraints: The list items must match regular expression `/(?!\\s)(?!.*\\s$)^(crn)[^'"<>{}\\s\\x00-\\x1F]*/`. The maximum length is `110` items. The minimum length is `0` items.
```
Comment (riccardo-angelilli) [inline website/docs/r/project_config.html.markdown:69]: this is not rendered well

Screenshot 2024-03-12 at 10 46 17

File: website/docs/r/project.html.markdown [hunk 4]
```diff
@@ -32,16 +32,16 @@ You can specify the following arguments for this resource.
 * `definition` - (Required, List) The definition of the project.
 Nested schema for **definition**:
 	* `description` - (Required, String) A brief explanation of the project's use in the configuration of a deployable architecture. It is possible to create a project without providing a description.
-	  * Constraints: The default value is ``. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
+	  * Constraints: The default value is `[]`. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
```
Comment (riccardo-angelilli) [inline website/docs/r/project.html.markdown:35]: the default value is not empty array `[]`, but empty string `''`

File: website/docs/r/project.html.markdown [hunk 5]
```diff
@@ -55,7 +55,7 @@ Nested schema for **configs**:
 	* `definition` - (List) The name and description of a project configuration.
 	Nested schema for **definition**:
 		* `description` - (String) A project configuration description.
-		  * Constraints: The default value is ``. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
+		  * Constraints: The default value is `[]`. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
```
Comment (riccardo-angelilli) [inline website/docs/r/project.html.markdown:58]: the default value is not empty array `[]`, but empty string `''`

File: website/docs/r/project.html.markdown [hunk 6]
```diff
@@ -102,21 +102,21 @@ Nested schema for **environments**:
 	* `definition` - (List) The environment definition used in the project collection.
 	Nested schema for **definition**:
 		* `description` - (String) The description of the environment.
-		  * Constraints: The default value is ``. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
+		  * Constraints: The default value is `[]`. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
```
Comment (riccardo-angelilli) [inline website/docs/r/project.html.markdown:105]: the default value is not empty array `[]`, but empty string `''`

File: website/docs/r/project_config.html.markdown [hunk 7]
```diff
@@ -55,9 +55,9 @@ Nested schema for **definition**:
 		* `instance_location` - (Optional, String) The location of the compliance instance.
 		  * Constraints: The maximum length is `12` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(us-south|us-east|eu-gb|eu-de)$/`.
 		* `profile_name` - (Optional, String) The name of the compliance profile.
-		  * Constraints: The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^(?!\\s)(?!.*\\s$)[^`<>\\x00-\\x1F]*$/`.
+		  * Constraints: The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^(?!\\s)(?!.*\\s$)[^<>\\x00-\\x1F]*$/`.
 	* `description` - (Optional, String) A project configuration description.
-	  * Constraints: The default value is ``. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
+	  * Constraints: The default value is `[]`. The maximum length is `1024` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s)(?!.*\\s$)[^\\x00-\\x1F]*$/`.
```
Comment (riccardo-angelilli) [inline website/docs/r/project_config.html.markdown:60]: the default value is not empty array `[]`, but empty string `''`

**Comment by h-romeus** [comment]
@riccardo-angelilli please take another look.

**Review by riccardo-angelilli — APPROVED** [review]
ok, now is fine

**Comment by riccardo-angelilli** [comment]
@h-romeus sync your fork with the remote main branch, then @hkantare can merge

**Comment by h-romeus** [comment]
image

I don't think I have the option to do so @riccardo-angelilli

**Comment by riccardo-angelilli** [comment]
ok, ready for merge

**Pull request merged**